test/unit/data-uri-link.html

```diff
@@ -0,0 +1,28 @@
+<!doctype html>
+<!--
+@license
+Copyright (c) 2017 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+<html>
+<head>
+  <script src="../../../webcomponentsjs/webcomponents-loader.js"></script>
+  <script src="../../../web-component-tester/browser.js"></script>
+  <link rel="import" href="../../polymer.html">
+  <link rel="import" href="data:text/html,%3Cdom-module%20id%3D%22element-data-uri%22%3E%3Cscript%3EPolymer(%7Bis%3A%22element-data-uri%22%2Cproperties%3A%7Bexists%3A%7Btype%3ABoolean%2Cvalue%3Atrue%7D%7D%7D)%3C%2Fscript%3E%3C%2Fdom-module%3E">
```

Comment:
Would it be possible to put this into code with the URLencode? That would make the test more readable and understandable.

Reply:
Possibly, yes. I can try at least :-)

Reply:
Updated; PTAL.

Reply:
Actually, what I attempted ([visible here](https://github.com/nicolasnoble/polymer/commit/b624a30ac92822c8332cce64af09eca4f7cae203)) works locally, but seems to timeout on tests on your travis environment. I'm not sure how to do this otherwise.